specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-09-01/virtualMachineScaleSet.json

```diff
@@ -2738,6 +2770,22 @@
             "type": "string",
             "description": "The instance ID of the virtual machine."
           },
+          {
```

Comment:
please add these at the end of the "parameters" to resolve the breaking change flag